pywbemtools/pywbemcli/_cmd_server.py

```diff
@@ -219,8 +220,55 @@ def server_remove_mof(context, **options):
 
 
 ###############################################################
-#         Server cmds
+#         Server cmds  Common components
+###############################################################
+
+
 ###############################################################
+#         Server cmds action functions
+###############################################################
+
+def cmd_server_namespaces(context):
+    """
+    Display namespaces in the current WBEM server
+    """
+    output_format = validate_output_format(
+        context.output_format,
+        ['TABLE', 'TEXT'], default_format=DEFAULT_TABLE_FORMAT)
+
+    try:
+        namespaces = context.wbem_server.namespaces
+        namespaces.sort()
+        context.spinner_stop()
+        if output_format_is_table(output_format):
+            # create list for each row
+            rows = [[ns] for ns in namespaces]
+
+            click.echo(format_table(rows, ['Namespace Name'],
+                                    title='Server Namespaces:',
+                                    table_format=output_format))
+        else:
+            display_text(", ".join(namespaces))
+
+    except Error as er:
+        raise click.ClickException('{}: {}'.format(er.__class__.__name__, er))
+
+
+def cmd_server_interop(context):
+    """
+    Display interop namespace in the current WBEM server
+    """
+
+    output_format = validate_output_format(context.output_format, 'TEXT')
+
+    try:
+        interop_ns = context.wbem_server.interop_ns
+        context.spinner_stop()
+
+        display_text(interop_ns, output_format)
+
+    except Error as er:
+        raise_pywbem_error_exception(er)
```

Comment:
That was partly because I started by building the statistics commands into the server group but then moved it to its own group but did not remove the code immediatly so the merging got mixed up.